src/index.ts

```diff
@@ -46,20 +41,62 @@ function displayNotification(
   }
 
   notificationPayload.body = message;
-  new Notification(`${notebookName}`, notificationPayload);
+  new Notification(title, notificationPayload);
+}
+
+/**
+ * Trigger notification.
+ */
+function triggerNotification(
+  cell: Cell,
+  notebook: Notebook,
+  cellStartTime: Date,
+  cellEndTime: Date,
+  minimumCellExecutionTime: number,
+  reportCellNumber: boolean,
+  reportCellExecutionTime: boolean,
+  cellNumberType: string,
+  failedExecution: boolean,
+  error: KernelError | null
+) {
+  const codeCell = cell.model.type === 'code';
+  const nonEmptyCell = cell.model.value.text.length > 0;
+  const codeCellModel = cell.model as ICodeCellModel;
+
+  if (codeCell && nonEmptyCell) {
+    const diff = new Date(<any>cellEndTime - <any>cellStartTime);
+    if (diff.getSeconds() >= minimumCellExecutionTime) {
```

Comment:
This makes the same issue in #16.

Reply:
Thanks for catching that! I think I resolved some of the merge conflicts incorrectly, so I'll make sure all the latest fixes are there.